core/commands/navigate.py

```diff
@@ -32,6 +32,11 @@ def run(self, edit, forward=True):
         sel.add(sublime.Region(new_position, new_position))
         self.view.run_command("show_at_center")
```

Comment:
The modern version of this is `self.view.show_at_center(new_position)`

Reply:
Following a rule for legacy code: Whenever you open a file to edit, leave it in a better state.

Reply:
Thats true, thanks